lib/auth.guard.ts

```diff
@@ -7,6 +7,7 @@ import {
   Optional,
   UnauthorizedException
 } from '@nestjs/common';
+import { GqlContextType, GqlExecutionContext } from '@nestjs/graphql';
```

Comment:
This will make the passport package require the GraphQL package regardless of GraphQL is used or not. This is not the approach we want to take, which is why it's left up to the developer to override the `getRequest` and `getResponse` methods _if needed_.